android/app/src/main/java/org/fossasia/openevent/fragments/AboutFragment.java

```diff
@@ -335,38 +336,18 @@ private void handleVisibility() {
 
     private void loadEventDates() {
         dateList.clear();
-        RealmResults<EventDates> eventDates = realmRepo.getEventDatesSync();
-        for (EventDates eventDate : eventDates) {
-            dateList.add(eventDate.getDate());
-        }
+        aboutFragmentViewModel.getDateList().observe(this, dateList -> {
+            this.dateList = dateList;
+        });
```

Comment:
Don't replace the instance. It is bad practice. The item should be as final as possible. Use `addAll` after clear. And clear after the data from viewmodel is loaded, i.e., inside the the lambda implementation